sdk/src/main/java/com/silanis/esl/sdk/examples/GenericFieldsExample.java

```diff
@@ -51,7 +51,7 @@
     public static final String DATEPICKER_ID = "datepickerId";
     public static final String DATEPICKER_NAME = "datepickerName";
     public static final int DATEPICKER_PAGE = 0;
-    public static final String DATEPICKER_VALUE = "datepickerValue";
+    public static final String DATEPICKER_VALUE = "05-15-2020";
```

Comment:
Please do not specify a specific fixed date. Because in the future, this date becomes too past. Therefore, specify a date that will fall within the scope of verification that may be added in the future.